packages/ui/react/src/components/Forms/Label/Label.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+import { Tooltip } from '../../Primitives'
+import { LabelElement, ContentFlex, QuestionIcon } from './Label.styled'
+import { getColor } from 'src/colors'
+
+type LabelProps = {
+  /** Main label text/node */
+  value: React.ReactNode
+  /** @deprecated if helpValue is present help is considered true */
+  help?: boolean
+  /** Help text shown in tooltip */
+  helpValue?: string
+  /** ARIA role */
+  role?: string
+  /** Adds "(Optional)" text */
```

Comment:
Can you apply the following suggestion?

This helps the prop description to not be directly tied to the UI effect, allowing for us to make changes in the future without needing to update the comment

```suggestion
  /**
   * Indicates that the field is optional
   * @default false
   */
```